### DISCUSSION TITLE
Sequence of runs for CAM7 development

### DISCUSSION THREAD

**Discussion by JulioTBacmeister** [NCAR/amwg_dev#193] [TF1: Unified physics for L58/L93]
@dan800 @bstephens82 @tilmes @andrewgettelman @cecilehannay @adamrher @Katetc @NCAR/amp 

This is a strawman breakdown for planning upcoming F-case simulations for CAM7 development

- Thread 1) non-orographic GW sensitivity at L58.  Use Cecile's current codebase, i.e., "26c configuration". This can be done quickly.  Not expecting perfect results, just looking for QBO-_like_ behavior in eq zonal wind.
- Thread 2) Implementing and tuning PUMAS with new CLUBB external (including prognostic momentum fluxes). Use Ben Stephen's current code base initially, then move to code w/ updated MAM. This has started.
- Thread 3) Implementing and tuning MAM5 (w/ mods to MAM4 when "simplified chemistry" is used?). Not sure how to proceed. Is it worth testing this in "26c config", or better to wait for progress in Thread 2?
- Thread 4) Implementing and testing new compsets/configurations, e.g., advected GHGs, full chemistry at both L58 and L93.

**Comment by tilmes** [comment]
Thread 3) is a bit more complicated, I would suggest this:
1. Implement and tune MAM4 (updated modes) in L58 without chemistry
2. Use this tuning to run MAM5 and implement and tune with chemistry in  L58.
4. Implement and tune MAM4 without chemistry (1), but now use oxidants from the chemistry, and also implement updated organic chemistry in the model

Other threads needed: 
a) test UBC with and without chemistry for L58
b) add new dust emissions (if available)
c) decide if marine organic aerosols should be added

**Reply by tilmes** [reply]
Thread 1): I would add some tuning depending on the aerosols (run diagnostics and decide)

**Comment by dan800** [comment]
@JulioTBacmeister You wrote:  To clarify: you are correct, we are planning to do F cases (or more exactly FWscHIST).

Can we get new compset defs made for B and F cases? This terminology is misleading and not very descriptive. F7L58HIST and B7L58HIST? i.e., CAM7 in L58 Historical F compset. Can make parallel compsets for the L93 higher top version.

**Reply by tilmes** [reply]
@JulioTBacmeister @dan800 For chemistry runs we would have to add a "C" in there. I think, adding a 7 for CAM7 does not make much sense to me, we have never done that for compsets, but adding level numbers or model top would be probably helpful.

**Reply by dan800** [reply]
My point remains - it's not W and I have said many times now SC doesn't really describe what is being done. If there's no ambiguity (i.e., only one set of CAM physics we can drop the 7).

**Reply by adamrher** [reply]
@dan800 I interpret "thread 4" as addressing the concerns you raised at the Friday meeting: define and name the workhorse compsets (prognostic GHG instead of "sc", verify no specified chemical heating rate, no non-LTE heating, specify H2O upper B.C.'s, GW param b.c.'s). So I consider FWscHIST as temporary until we get better compset name via thread 4. Or is your concern FWscHIST is wrong (not the name, but the configuration) to be using to address threads 1-3, even as Cecile dresses it up with namelists and sourcemods as we proceed through threads 1-4?

**Reply by JulioTBacmeister** [reply]
I like adding the level qualifiers.  I think putting the '7' after the F and B is succinct, but I worry that it is confusing because it addresses only the CAM version and not the land, ocean or other components being used.  I think we should dispense with 'W' and 'sc' qualifiers ASAP.   

If I understand @dan800 we don't need to worry about adding any extra qualifiers to the compset that indicate prognostic GHG since this will now be the default for both L58 and L93 unless full chemistry is used.  Do we need any extra qualifiers to mark full interactive chemistry, whatever that may be in L58 and L93?

So dropping the '7' for now, new F-case compsets would be for example:
FL58HIST, FL93HIST (both using prognostic GHGs)
FL58HISTchem, FL93HISTchem (both using some full interactive chemistry)

**Reply by tilmes** [reply]
For chemistry, we usually add the C after the F, so either FCHISTL58 (and FHISTL58), or FCL58HIST
Simone

**Reply by dan800** [reply]
Based on the discussion above, I'd go with:
FL58HIST, FL93HIST (both using prognostic GHGs)
FCL58HIST, FCL93HIST (both using some form of full interactive chemistry)

I know there's still lots to do but how hard is it to make new compset defs in the development trunk? The sooner we adopt a compset name, which is carried through to case names, the sooner all our plot titles will be more useful in diagnostics.

**Reply by adamrher** [reply]
IIRC Cecile is still using the long name compset definition and just writing in "FWscHIST" to the case names for all our runs. So she could probably just start using this new naming convention ASAP?

**Reply by dan800** [reply]
_So she could probably just start using this new naming convention ASAP?_
That would be great.

**Comment by adamrher** [comment]
The CTSM folks would like us to update to CLM5.1 in our compset definitions (see discussion here: https://github.com/NCAR/amwg_dev/discussions/196#discussioncomment-3994557). They'd request we do this sooner rather than later, in case there are issues with the new external. Cecile's current cesm3 "sandbox" does support CLM5.1. I'd suggest we switch over during this F-case phase.
